src/js/components/RadioButtonGroup/stories/CustomTheme.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Grommet, RadioButtonGroup, ThemeContext } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+
+const customTheme = deepMerge(grommet, {
+  radioButton: {
+    border: {
+      color: 'red',
+      width: '10px',
+    },
+    hover: {
+      border: {
+        color: 'blue',
+      },
+    },
+    size: '100px', // affects the size of the outer circle
```

Comment:
Can we add these comments to the themeDoc? src/js/components/RadioButton/doc.js. could be very helpful.

Reply:
will do